### ISSUE TITLE
Config not being generated 

### ISSUE THREAD

**Issue by dymattic** [sdaqo/anipy-cli#114]
**Describe the bug**
For new installs the config should be generated and pre-filled with default values based on the Config-Class' @Property fields.
For some reason this doesn't happen.

Even if no config-file exists the config is not "none" when reading from file in:

[config.py :14](https://github.com/sdaqo/anipy-cli/blob/cccf65144cf96d92c8e30fd2584353ae5f0e3c37/anipy_cli/config.py#L14)
```python
with self._config_file.open("r") as conf:
    self._yaml_conf = yaml.safe_load(conf)
    if self._yaml_conf is None:
        # The config file is empty
        self._yaml_conf = {}
```

Checked on Windows11 only so far.

**Comment by sdaqo** [comment]
this looks like a windows problem, linux works fine

**Comment by dymattic** [comment]
alright, thanks for checking. I'll look into this

**Comment by ivanvx1** [comment]
Also for macos, the default config file is not created but works if done by the user

**Comment by DragonOfShuu** [comment]
I'm on Windows 11. I'll go ahead and see what I can find out.

**Comment by DragonOfShuu** [comment]
# TL;DR
The current version of anipy doesn't create a config file. But when I force it to, it creates it, and reads it, but I cannot find it anywhere on the file system.

# Breakdown
According to #79, the config system was changed so users don't have to manually add vars every update. Great change, but by doing so they also removed the code that creates at least some kind of default file we can find and adjust (`_create_config()` has no references).

To fix this, I added the code that creates the config file, but there is indeed an issue on Windows (I've only test Windows). Python says that the config file was created, it pulls data from it and everything, but I can't actually find the file on my system. 

Pulled data evidence:
<img width="368" alt="image" src="https://github.com/sdaqo/anipy-cli/assets/68718280/7aec340e-9589-421d-9978-d646eafe191b">

Can't find it:
<img width="318" alt="image" src="https://github.com/sdaqo/anipy-cli/assets/68718280/72026683-2b13-4d6c-a29d-58f1db48f5cb">

I went through the computer's process of placing the config file, and I still couldn't find it no matter what I did. 
I found a Stack Overflow question about it, but there were no answers.

**Comment by DragonOfShuu** [comment]
# Update:

When using different Python versions, the one that created the file seems to recognize it and the other does not

<img width="1139" alt="image" src="https://github.com/sdaqo/anipy-cli/assets/68718280/8c793f50-09e8-470e-a463-3dc93a561914">

**Comment by DragonOfShuu** [comment]
# GOOD NEWS

I found where Python is actually writing the file. Turns out if you download Python from the Microsoft store, Python gets sandboxed, and anything written to AppData is written into its cache, as seen here:

<img width="470" alt="image" src="https://github.com/sdaqo/anipy-cli/assets/68718280/2f60fc3c-f2e6-4632-8ceb-adf365dbd518">

Unfortunately there aren't many good solutions for solving this, except for knowing that your Python might be sandboxed. So, for now I'll just give you the possible location your config file might get stored on a sandboxed Python on Windows:

`%USERPROFILE%\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\Local\anipy-cli\config.yaml`

I recommend either users download Python directly from the site, or be warned that Microsoft Store Python is sandboxed and will be stored in this location. You could also relocate where the config file is placed if you wish, but I'm not sure where it could go.

**Comment by sdaqo** [comment]
Thanks for the investigation, will close this  as #136 (2.7.23) is now merged.
